src/main/osd/osd_elements.c

```diff
@@ -922,6 +923,26 @@ static void osdElementGpsSpeed(osdElementParms_t *element)
 {
     tfp_sprintf(element->buff, "%c%3d%c", SYM_SPEED, osdGetSpeedToSelectedUnit(gpsConfig()->gps_use_3d_speed ? gpsSol.speed3d : gpsSol.groundSpeed), osdGetSpeedToSelectedUnitSymbol());
 }
+
+static void osdElementEfficiency(osdElementParms_t *element)
+{
+    int efficiency = 0;
+    if (sensors(SENSOR_GPS) && ARMING_FLAG(ARMED) && STATE(GPS_FIX) && gpsSol.groundSpeed >= EFFICIENCY_MINIMUM_SPEED_CM_S) {
+        const int speedX10 = osdGetSpeedToSelectedUnit(gpsSol.groundSpeed * 100); // speed * 100 for improved resolution at slow speeds
+        
+        if (speedX10 > 0) {
+            const int mAmperage = getAmperage() * 10; // Current in mA
+            efficiency = mAmperage * 100 / speedX10; // mAmperage * 100 to cancel out speed * 100 from above
```

Comment:
I don't really see how multiplying both the numerator and denominator by 100 helps improve resolution in integer math. The loss of resolution comes from converting centimeters/second to km/h or mph before the calculation.

Reply:
The `* 100` is applied to the speed in cm/s (`gpsSol.groundSpeed`) _before_ conversion to km/h or mph.

Reply:
Ok, sorry I missed that it was before.